Client/include/client.h

```diff
@@ -10,7 +10,11 @@
 
 class ChronoLogClient {
 public:
-    ChronoLogClient() { CHRONOLOG_CONF->ConfigureDefaultClient("../../test/communication/server_list"); }
+    ChronoLogClient() //{ CHRONOLOG_CONF->ConfigureDefaultClient("../../test/communication/server_list"); }
+    {
+	CHRONOLOG_CONF->IS_SERVER = false;
+	adminRpcProxy_ = ChronoLog::Singleton<ChronoLogAdminRPCProxy>::GetInstance();
```

Comment:
ChronoLogAdminRPCProxy constructor calls Configuration_Manager during Get_Instance() call. This is the only call to Configuration_Manager in this program. I have removed it from the default Client Constructor. I did not want to modify the constructors of other classes since there is a separate issue to decouple the RPC instance from the Client constructor and also the combine adminRPC and metaDataRPC.